### PR TITLE
:sparkles: Get pod list

### PR DESCRIPTION
**Pull request by shawn-hurley**
When we are failing to run tasks, getting the pods will help determine what is happening.

### CODE REVIEW

File: .github/workflows/main-tier0.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
         uses: konveyor/tackle2-operator/.github/actions/install-tackle@main
       - uses: actions/setup-go@v3
         with:
-          go-version: '1.19'
+          go-version: '1.22'
```
Comment (aufi) [inline .github/workflows/main-tier0.yml:26]: No big deal, but I'd prefer use the same go versions for on-PR and nightly-executed workflows.

File: analysis/analysis_test.go [hunk 2]
```diff
@@ -17,10 +18,19 @@ import (
...
 )
 
 // Test application analysis
 func TestApplicationAnalysis(t *testing.T) {
+
+	// Create a kubernetes client from the default kubeconfig
+	cl, err := client.New(config.GetConfigOrDie(), client.Options{})
+	if err != nil {
+		t.Fatalf("failed to create client: %v", err)
+	}
```
Comment (aufi) [inline analysis/analysis_test.go:33]: I'd suggest make this optional (use an env variable like KUBEDEBUG similar to https://github.com/konveyor/go-konveyor-tests?tab=readme-ov-file#test-execution-options, or just skip if client creation failed).

I'm sure it brings value for debugging, but not each test execution (e.g. local or jenkins) has available KUBECONFIG.

**Review by aufi — COMMENTED** [review]
Thanks for PR, looks good to me, just added few comments.

File: analysis/analysis_test.go (hunk 2 above)
Comment (shawn-hurley) [reply analysis/analysis_test.go:33]: That is good point, will make the change!

File: .github/workflows/main-tier0.yml (hunk 1 above)
Comment (shawn-hurley) [reply .github/workflows/main-tier0.yml:26]: Yeah this was a change that had to be reverted sorry about leaving that in

**Pull request closed without merging**